### DISCUSSION TITLE
React Native Expo Production App Not Getting data

### DISCUSSION THREAD

**Discussion by Hannes1** [supabase/supabase#1697] [Questions] [answered]
Hi, has anyone tried to use supabse inside a  react native production build?
If so is supabase working, because when developing supabase works perfectly but in production awaiting supabase query never gets resolved, has anyone else experienced this?

**Comment by soedirgo** [comment, marked as answer]
Related: https://github.com/supabase/supabase/issues/1693. Is this the same issue you're experiencing @Hannes1?

**Reply by Hannes1** [reply]
Indeed thanks, will follow the issue!
